strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/repository/NpmRepositoryFeatures.java

```diff
@@ -500,21 +501,23 @@ public void handle(RemoteRepositorySearchEvent event)
 
     }
 
-    private Long countPackages(String storageId,
-                               String repositoryId,
-                               Predicate predicate)
+    public Boolean anyRemotePackageCached(String storageId,
+                                                 String repositoryId)
```

Comment:
Cosmetic: Please, re-align the method parameters.

Reply:
Fixed